.gitignore

```diff
@@ -15,9 +15,9 @@
 .env.development.local
 .env.test.local
 .env.production.local
-.env.slack-api
-.env.transit-api
-.env.github-api
+.env.slack
```

Comment:
@SSabu Please make sure the .gitgnore file you have here is exactly the same as upstream/dev. This would also fix the issue of the addition of the .env files I see at the top here

Reply:
I meant @Dave-Gallup. But this goes for @SSabu as well